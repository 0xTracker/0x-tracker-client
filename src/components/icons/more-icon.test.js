import { render } from 'react-testing-library';
import React from 'react';

import MoreIcon from './more-icon';

describe('more icon component', () => {
  it('should render without props', () => {
    const { container } = render(<MoreIcon />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
