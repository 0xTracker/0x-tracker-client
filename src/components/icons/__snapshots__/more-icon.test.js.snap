// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`more icon component should render without props 1`] = `
.c0 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

<svg
  aria-hidden="true"
  class="c0"
  fill="currentColor"
  focusable="false"
  viewBox="0 0 24 24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M0 0h24v24H0z"
    fill="none"
  />
  <path
    d="M6 10c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm12 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-6 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
  />
</svg>
`;
